build/parser.js

```diff
@@ -403,13 +403,13 @@ class Parser {
         else if (item.slot === 5) item.category = 'Melee'
         else if (item.slot === 0) item.category = 'Secondary'
         else if (item.slot === 1) item.category = 'Primary'
+        else if (item.uniqueName.includes('/MoaPetHead')) item.category = 'Pets'
```

Comment:
this should be a reorg in the types parser, preferrably

Reply:
instead, add `MoaPetHead` as a key that gets the `Pet` type in [item-types.json](https://github.com/WFCD/warframe-items/blob/development/config/itemTypes.json)